<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1172</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="addParBtn">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>90</y>
      <width>94</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Paragraph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endParBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>90</y>
      <width>94</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>End Paragraph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endHeadBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>50</y>
      <width>94</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>94</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>End Header</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contBtn">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>330</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add context</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterBtn">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>250</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter Button</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addFntBtn">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>130</y>
      <width>94</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>94</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Font edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endFntBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>130</y>
      <width>94</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>94</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>End Font edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addHeadBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>50</y>
      <width>94</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>94</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Add Header</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endTitBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>170</y>
      <width>94</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>94</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>End Title</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addTitBtn">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>170</y>
      <width>94</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>94</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Add Title</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addBrBtn">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>210</y>
      <width>94</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>94</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>&lt;br&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addHrBtn">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>210</y>
      <width>94</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>94</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>&lt;hr&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>94</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>94</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>10</y>
      <width>94</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>94</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>End</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtbox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>841</width>
      <height>441</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="insBox">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>250</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>50</y>
      <width>46</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>6</number>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
